README.md

```diff
@@ -29,6 +29,27 @@ const profile = await liff.getProfile();
 console.log(profile);
 ```
 
+### With [Pluggable SDK](https://developers.line.biz/en/docs/liff/pluggable-sdk/)
+
```

Comment:
```suggestion

If you use LIFF Mock with Pluggable SDK mode, you have to install `IsInClientModule` before the installation of `LiffMockPlugin` because LIFF Mock depends on `liff.isInClient` API.

```